D_controllers.md

```diff
@@ -22,7 +24,7 @@ In addition, the `PageController` gives us the `index` action to display the Pho
 ###Actions
 Controller actions are just functions. We can name them anything we like as long as they follow Elixir's naming rules. The only requirement we must fulfill is that the action name matches a route defined in the router.
 
-For example, we could change the action name in the default route that Phoenix gives us in a new app from index:
+For example, in `web/router.ex` we could change the action name in the default route that Phoenix gives us in a different app than index, for example test:
```

Comment:
The original is correct in this case. ":index" isn't the app name, it's the name of the action.